tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -205,6 +207,53 @@ def test_aggregate_function_count(self):
         assert response.status_code == 200, response.content
         assert [attrs for time, attrs in response.data["data"]] == [[{"count": 1}], [{"count": 2}]]
 
+    def test_aggregate_function_apdex(self):
+        project1 = self.create_project()
+        project2 = self.create_project()
+
+        ProjectTransactionThreshold.objects.create(
+            project=project1,
+            organization=project1.organization,
+            threshold=100,
+            metric=TransactionMetric.DURATION.value,
+        )
+
+        events = [
+            ("one", 400, project1.id),
+            ("one", 400, project1.id),
+            ("two", 3000, project2.id),
+            ("two", 1000, project2.id),
+            ("three", 3000, project2.id),
+        ]
+        for idx, event in enumerate(events):
+            data = load_data(
+                "transaction",
+                start_timestamp=self.day_ago + timedelta(minutes=(1 + idx)),
+                timestamp=self.day_ago + timedelta(minutes=(1 + idx), milliseconds=event[1]),
+            )
+            data["event_id"] = f"{idx}" * 32
+            data["transaction"] = f"/apdex/new/{event[0]}"
+            data["user"] = {"email": f"{idx}@example.com"}
+            self.store_event(data, project_id=event[2])
+
+        with self.feature("organizations:discover-basic"):
+            response = self.client.get(
```

Comment:
`sentry.utils.snuba.QueryExecutionError: [215] DB::Exception: Column `project_id` is not under aggregate function and not in GROUP BY. Stack trace:`